source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,48 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.12.0:
+
+{+atlas-cli+} 1.12.0
+--------------------
+
+- Drops support for Ubuntu 18.04.
+- Adds :ref:`atlas-deployments` commands so that you can work with
+  |service|, including using |fts| and {+avs+}, throughout the entire 
+  software development lifecycle, from your local environment to the 
+  cloud. To learn more, see :ref:`atlas-cli-local-cloud`.
+- Adds the following new commands to manage {+atlas-sp+} instances
+  and connection registries:
+
+  - :ref:`atlas-streams-instances-create`
+  - :ref:`atlas-streams-instances-delete`
+  - :ref:`atlas-streams-instances-describe`
+  - :ref:`atlas-streams-instances-list`
+  - :ref:`atlas-streams-instances-update`
+  - :ref:`atlas-streams-connections-create`
+  - :ref:`atlas-streams-connections-delete`
+  - :ref:`atlas-streams-connections-describe`
+  - :ref:`atlas-streams-connections-list`
+  - :ref:`atlas-streams-connections-update`
+
+{+atlas-cli+} 1.11.0
+--------------------
+
+- Fixes an issue that occurred when you created x509 certs for database 
+  users. The command didn't output the certificate. If you depend on 
+  the previously incorrect output, this update is a breaking change.
+
+- :ref:`atlas-alerts-settings` create and update commands perform 
+  validation that helps you formulate all required arguments per each 
```

Comment:
Nit: "for each" is more commonly used than "per each" so will improve reader comprehension
```suggestion
  validation that helps you formulate all required arguments for each 
```